src/config/CodeGeneratorConfig.php

```diff
@@ -4,9 +4,9 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CodeGeneratorConfig {
-	
+
 	protected $options;
-	
+
 	public function __construct(array $options = []) {
```

Comment:
So I implemented this now. The only thing now that I am a little worried about is that specifying a type for a constructor or destructor will break since they aren't allowed to have a return type.